app/public/stylesheet/dashboard.css

```diff
@@ -0,0 +1,53 @@
+.dashboard_page{
+    margin: 0;
+    padding: 0;
+    background-color:rgb(250, 250, 250);
+    width:100%;
+    height:100%;
+    overflow: hidden;
+}
+
+.container{
+    width: 100% !important;
+}
+
+.inner_logo{
+    position: relative;
+    top: 1vh;
+}
+
+.sign_out_button{
```

Comment:
To vertically align your button would it work to add display: inline-block; and vertical-align: middle? Similarly for the dropdown selectors.

Reply:
Been resolved via other methods.